briefcase/android.py

```diff
@@ -107,11 +108,18 @@ def post_install(self):
     def build_app(self):
         if not self.start:
             print(" * Building %s..." % (self.formal_name))
-            proc = subprocess.Popen([
-                    './gradlew', 'build'
+            if sys.platform == 'win32':
+                proc = subprocess.Popen([
+                    'cmd.exe', '/c', 'gradlew.bat', 'build'
                 ],
-                cwd=os.path.abspath(self.dir)
-            )
+                    cwd=os.path.abspath(self.dir)
+                )
+            else:
+                proc = subprocess.Popen([
+                        '/usr/bin/env', 'sh', './gradlew', 'build'
```

Comment:
Is the `/usr/bin/env sh` bit really needed?

Reply:
I felt it was safer than assuming the shell script had executable permissions, and also matched the windows cmdline better by declaring the shell it's running in.